<launch>
    <!--Control-->
    <arg name="max_linear_velocity" value="0.25" />
    <arg name="max_angular_velocity" value="1.57079632679" />
    <node pkg="joy" type="joy_node" name="joy_node"/>
    <node pkg="vistech" type="autonomous_follow_closest_object.py" name="autonomous_follow_closest_object">
        <param name="max_linear_velocity" value="$(arg max_linear_velocity)" />
        <param name="max_angular_velocity" value="$(arg max_angular_velocity)" />
    </node>
    <node pkg="vistech" type="autonomous_traffic_signs.py" name="autonomous_traffic_signs">
        <param name="max_linear_velocity" value="$(arg max_linear_velocity)" />
        <param name="max_angular_velocity" value="$(arg max_angular_velocity)" />
    </node>
    <node pkg="vistech" type="control.py" name="control">
        <param name="max_linear_velocity" value="$(arg max_linear_velocity)" />
        <param name="max_angular_velocity" value="$(arg max_angular_velocity)" />
    </node>

    <!--Motors-->
    <include file="$(find vistech)/launch/roboclaw_ros/roboclaw.launch">
        <arg name="address_master" default="128"/>
        <arg name="address_slave" default="129"/>
    </include>

</launch>